### PR TITLE
Add solution to exercise 'calculate'

### PR DESCRIPTION
**Pull request by MoreThanADC**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @MoreThanADC

**Comment by ziobron** [comment]
⏱ 3 XP extra for @MoreThanADC for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,19 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add") {
+        return std::to_string(first + second);
+    } else if (command == "subtract") {
+        return std::to_string(first - second);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:8]: If you return from if statement, you shouldn't use if else. This is redundant.

File: homework/calculate/test.cpp [hunk 2]
```diff
@@ -6,27 +6,31 @@ TEST(calculate, ShouldAdd) {
   EXPECT_EQ(calculate("add", 15, 5), "20");
   EXPECT_EQ(calculate("add", 0, 0), "0");
   EXPECT_EQ(calculate("add", 1, -20), "-19");
+  EXPECT_EQ(calculate("add", -3, -7), "-10");
```
Comment (nauka-programowania-MA) [inline homework/calculate/test.cpp:9]: Nice :) It's good that you handle case for negate values!

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**